gui/ThemeLayout.cpp

```diff
@@ -242,6 +242,14 @@ void ThemeLayoutMain::reflowLayout(Widget *widgetChain) {
 		}
 	}
 
+	if (g_gui.useRTL()) {
+		if (this->_name == "GameOptions" || this->_name == "GlobalOptions") {
```

Comment:
As discussed, this is not a good idea to hardcode names like this. What if you compare the dialog width with the overlay width?

Reply:
So, I looked into this and your idea is good, but it doesn't work here.

Here, I am trying to set the dialog paddings _only_ when I encounter an entire dialog. `ThemeLayoutMain::reflowLayout()` is being called many times for the widgets that are present inside the dialog menu, thus disturbing and breaking other things.

Eg - Once "GameOptions" is being called, then widgets and containers inside like "GameOptions_Graphics", etc are also being called. dialog width and overlay width are different for them, so they also fall true against the condition you suggested - breaking my intention.